src/Controls/src/Core/Platform/Windows/TimePickerExtensions.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Microsoft.Maui.Controls.Platform
+{
+	public static class TimePickerExtensions
+	{
+		public static void UpdateText(this UI.Xaml.Controls.TimePicker platformTimePicker, ITimePicker timePicker, TextTransform tranform)
+		{
+			// TODO: Implement TextTransform property on Windows.
```

Comment:
Has this property been implemented in Xamarin.Forms?, What should we do on Windows?

![image](https://user-images.githubusercontent.com/6755973/156175891-3c67fd60-663d-4867-80d7-8ee65dbb2d35.png)

We can access the elements that make up the control, and apply the transformation to the text of each TextBlock. Opinions?
```
<Border x:Name="FirstPickerHost" Grid.Column="0">
    <TextBlock x:Name="HourTextBlock" AutomationProperties.AccessibilityView="Raw" FontFamily="{TemplateBinding FontFamily}" FontWeight="{TemplateBinding FontWeight}" FontSize="{TemplateBinding FontSize}" Padding="{ThemeResource TimePickerFlyoutPresenterItemPadding}" TextAlignment="Center"/>
</Border>
<Rectangle x:Name="FirstColumnDivider" Grid.Column="1" Fill="{ThemeResource TimePickerSpacerFill}" HorizontalAlignment="Center" Width="2"/>
<Border x:Name="SecondPickerHost" Grid.Column="2">
    <TextBlock x:Name="MinuteTextBlock" AutomationProperties.AccessibilityView="Raw" FontFamily="{TemplateBinding FontFamily}" FontWeight="{TemplateBinding FontWeight}" FontSize="{TemplateBinding FontSize}" Padding="{ThemeResource TimePickerFlyoutPresenterItemPadding}" TextAlignment="Center"/>
</Border>
<Rectangle x:Name="SecondColumnDivider" Grid.Column="3" Fill="{ThemeResource TimePickerSpacerFill}" HorizontalAlignment="Center" Width="2"/>
<Border x:Name="ThirdPickerHost" Grid.Column="4">
    <TextBlock x:Name="PeriodTextBlock" AutomationProperties.AccessibilityView="Raw" FontFamily="{TemplateBinding FontFamily}" FontWeight="{TemplateBinding FontWeight}" FontSize="{TemplateBinding FontSize}" Padding="{ThemeResource TimePickerFlyoutPresenterItemPadding}" TextAlignment="Center"/>
</Border>
```